codec-http2/src/test/java/io/netty/handler/codec/http2/Http2ConnectionHandlerTest.java

```diff
@@ -124,7 +125,7 @@
     public void setup() throws Exception {
         MockitoAnnotations.initMocks(this);
 
-        promise = new DefaultChannelPromise(channel);
+        promise = new DefaultChannelPromise(channel, ImmediateEventExecutor.INSTANCE);
```

Comment:
channel is a mock object and the `eventLoop()` was not mocked ... so it was returning `null` ... it is not a valid state for the promise to have a `null` executor when the promise is completed.